### ISSUE TITLE
Pre-built binaries missing in newest releases

### ISSUE THREAD

**Issue by theguy147** [gobuffalo/buffalo#2078]
As stated in the title the pre-built binaries (incl checksums) are missing in the latest releases (v0.16.19 & v0.16.20). Not sure if this is intended.

Therefore also the install instructions on https://gobuffalo.io/en/docs/getting-started/installation are not up-to-date.

**Comment by saurori** [comment]
Also the [Buffalo Homebrew Tap](https://github.com/gobuffalo/homebrew-tap/tree/master) has not been updated.

**Comment by acaloiaro** [comment]
@theguy147 I submitted #2079 to address this issue. 

The first failure can be seen here https://github.com/gobuffalo/buffalo/runs/1665725191.

```
⨯ release failed after 0.00s error=yaml: unmarshal errors:
  line 41: field github not found in type config.Homebrew
```

This was due to a deprecation in `goreleaser`: `github` was superseded by `tap`.

**Comment by paganotoni** [comment]
Just released v0.16.21 with the fix. thanks @acaloiaro for it.
